ios/RNBLEPrinter.h

```diff
@@ -15,13 +14,11 @@
 #import <React/RCTBridgeModule.h>
 #endif
 #import <CoreBluetooth/CoreBluetooth.h>
+#import "PrinterSDK.h"
 
 @interface RNBLEPrinter : NSObject <RCTBridgeModule>{
-    NSMutableArray* _printerArray;
-    NSObject* m_printer;
+    Epos2Printer *printer_;
```

Comment:
why to call it printer_, just call it printer

Reply:
Updated. in the next pr.